CHANGELOG.md

```diff
@@ -1,3 +1,302 @@
+
+##### Old:
+
+```hcl
+packet {
+  oidc {
+    client_id = "gangway"
+  }
+}
+```
+
+##### New:
+
+```hcl
+packet {
+  oidc {
+    client_id = "clusterauth"
+  }
+}
+```
+
+#### Cluster update steps
+
+Ensure your cluster is in a healthy state by running `lokoctl cluster apply` using the`v0.4.1` version. Updating multiple versions at a time is not supported so, if your cluster is older, upgrade to `v0.4.1` and only then proceed with the upgrade to `v0.5.0`.
+
+Due to [Terraform](https://github.com/kinvolk/lokomotive/pull/824) and [Kubernetes](https://github.com/kinvolk/lokomotive/pull/1030) updates to v0.13+ and v1.19.3 respectively, some manual steps need to be performed when updating. In your cluster configuration directory, follow these steps:
+
+1. Update local Terraform binary to version v0.13.X. You can follow [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli) to do that.
+
+2. Starting from your cluster directory, export your platform name and assets directory name used in your platform configuration. It will be used in next steps:
+  ```sh
+  export PLATFORM="packet" && export ASSETS_DIR="assets"
+  ```
+
+3. Remove old asset files:
+  ```sh
+  rm -f $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/require.tf \
+  $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/workers/require.tf \
+  $ASSETS_DIR/terraform-modules/dns/route53/require.tf
+  ```
+
+4. Go to the `terraform` directory:
+  ```sh
+  cd $ASSETS_DIR/terraform
+  ```
+
+5. Replace the old providers:
+  ```sh
+  terraform state replace-provider -auto-approve registry.terraform.io/-/ct registry.terraform.io/poseidon/ct && \
+  terraform state replace-provider -auto-approve registry.terraform.io/-/template registry.terraform.io/hashicorp/template
+  ```
+
+6. Return to original directory and use kubeconfig generated by lokomotive:
+
+  ```sh
+  cd - && export KUBECONFIG=$ASSETS_DIR/cluster-assets/auth/kubeconfig
+  ```
+
+7. `FelixConfiguration` has been moved to calico charts. To avoid firewall interruption, label and annotate it so that it can be managed by Helm while updating:
+  ```sh
+  kubectl label FelixConfiguration default app.kubernetes.io/managed-by=Helm --overwrite=true && \
+  kubectl annotate FelixConfiguration default meta.helm.sh/release-name=calico --overwrite=true && \
+  kubectl annotate FelixConfiguration default meta.helm.sh/release-namespace=kube-system --overwrite=true
+  ```
+
+Finally, run the following:
+
+```sh
+lokoctl cluster apply --skip-components -v
+```
+
+**NOTE:** On clusters with a single controller node only, delete the old `kube-apiserver` ReplicaSet during cluster update. It deletes old ReplicaSet which is replaced by new one during cluster update:
```

Comment:
I'd mention here, then lokoctl prints that kube-apiserver is being updated, as running the command before doesn't really serve the purpose.